src/client/chowly-api-client.ts

```diff
@@ -24,9 +24,25 @@ export class ChowlyApiClient {
         });
     }
 
-    async list(): Promise<Sections> {
+    async getMenu(): Promise<Sections> {
         const response: AxiosResponse<ISections> = await this.client.get<ISections>(`${this.basePrefix}/menus`);
 
         return plainToClass(Sections, response.data);
     }
+
+    async createOrder(order: ICreateOrder): Promise<Order> {
+        const response: AxiosResponse<ICreateOrder> = await this.client.post<ICreateOrder>(`${this.basePrefix}/orders`, order, {
+            headers: { 'Content-Type': 'application/json' },
```

Comment:
why this method have `headers` but `getMenu` is missing them?

Reply:
getMenu working without headers, but create Order and get Order not working without headers ( check in postman).